source/k64f_eth_nanostack_port.c

```diff
@@ -668,6 +667,7 @@ void ENET_Receive_IRQHandler(void)
     enet_dev_if_t *ethernet_iface_ptr =
             &ethernet_iface[BOARD_DEBUG_ENET_INSTANCE];
 
+    platform_interrupts_disabled();
```

Comment:
What is this supposed to do?
If the functions between `platform_interrupts_disabled()` and `platform_interrupts_enabling()` don't call enter/exit critical, this does not make sense.